tests/python_tests/test_kv_cache_eviction.py

```diff
@@ -294,12 +299,13 @@ def test_optimized_generation_longbench(test_struct):
     assert avg_optimization_ratio >= test_struct.avg_cache_usage_optimization_ratio
 
 
-@pytest.mark.nightly
+@pytest.mark.precommit
 @pytest.mark.parametrize("device", ["CPU"])
 @pytest.mark.parametrize("test_struct", [
-    KVCrushTestData(subset="samsum"),
-    KVCrushTestData(subset="trec"),
-], ids=["samsum", "trec"])
+    KVCrushTestData(subset="samsum", test_mode="optimal"),
```

Comment:
Do you really need KVCrushTestData? I think it can be replaced to `@pytest.mark.parametrize("subset", [...])`